pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py

```diff
@@ -335,9 +270,13 @@ def __init__(self, data, mount_points):
         self._warningLabel = self.builder.get_object("mountPointWarningLabel")
 
     def on_add_confirm_clicked(self, button, *args):
-        self.mountpoint = self.builder.get_object("addMountPointEntry").get_active_text()
-        self._error = validate_mountpoint(self.mountpoint, self.mountpoints,
-                                          strict=False)
+        self.mount_point = self.builder.get_object("addMountPointEntry").get_active_text()
+
+        if lowerASCII(self.mount_point) in ("swap", "biosboot", "prepboot"):
```

Comment:
I wonder if this might be a good candidate for a constant holding a list/tuple ?

Reply:
I will keep it in my mind for future improvements.